integration-tests/dynamic-config-tests/src/test/resources/microservice.yaml

```diff
@@ -23,15 +23,11 @@ service_description:
 
 apollo:
   config:
-    serverUri: http://apollo.servicecomb.apache.org:8070
+    serverUri: http://localhost:8080
```

Comment:
it's better to use another not normal port, that's make developers run the case in IDE easier.

Reply:
Done